### PR TITLE
feat: capitalize first letter

### PR DESCRIPTION
**Pull request by alanneves**
in this pr i created the `capitalizeFirstLetter` function

### CODE REVIEW

**Review by Viserion77 — DISMISSED** [review]
LGTM

File: __tests__/string/formatters/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { capitalizeFirstLetter } from '../../../src/string'
+
+describe('capitalizeFirstLetter', () => {
+  const data = [{input: 'teste', output: 'Teste'}, {input: 'TESTE', output: 'TESTE'}, {input: 'alan Reno Neves', output: 'Alan Reno Neves'}]
+
+  test.each(data)('Should return true if parameter is string', (param) => {
```
Comment (mickaelnn) [inline __tests__/string/formatters/index.test.ts:6]: This description is correct?

**Review by mickaelnn — CHANGES_REQUESTED** [review]

**Review by mickaelnn — DISMISSED** [review]
LGTM

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by planivic1 — APPROVED** [review]
LGTM

**Pull request merged**